spec/models/spree/printables/order/invoice_view_spec.rb

```diff
@@ -0,0 +1,72 @@
+RSpec.describe Spree::Printables::Order::InvoiceView do
+  let(:invoiceable) { Spree::Order.new }
+  let(:invoice_view) { Spree::Printables::Order::InvoiceView.new(invoiceable) }
+
+  describe "#bill_address" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.